### PR TITLE
ignore auto generated clion files

### PR DESCRIPTION
**Pull request by superbiebel**
ignore auto generated cmake-build-debug folder (default build directory)
ignore auto generated .idea directory (user specific settings)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@
 /build/
 /out/
 bazel-*
-CMakeSettings.json
\ No newline at end of file
+CMakeSettings.json
+/cmake-build-debug/
```
Comment (ben-clayton) [inline .gitignore:7]: A very minor comment:
I'm not familiar with clion, but I'm guessing there might be a `cmake-build-release` directory if you build a non debug build?
If so, would this be better as `/cmake-build-*/` ?
Would also be nice if this were placed between `/build/` and `/out/`.
Happy if you want to fix this as a follow-up.

**Review by ben-clayton — COMMENTED** [review]
Thank you for your PR. One minor comment, otherwise LGTM.
I'll merge tomorrow with or without follow-up changes.

File: .gitignore (hunk 1 above)
Comment (superbiebel) [reply .gitignore]: fixed

**Review by ben-clayton — APPROVED** [review]
Thank you!

**Pull request merged**